src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double upperSide;
+    private double lowerSide;
+    private double height;
+
+    public IsoscelesTrapezoid(String color, double upperSide, double lowerSide, double height) {
+        super(color);
+        this.upperSide = upperSide;
+        this.lowerSide = lowerSide;
+        this.height = height;
+    }
+
+    @Override
+    public double calculateArea() {
+        return (double) Math.round(((upperSide + lowerSide) / 2) * height) * 10 / 10;
```

Comment:
Avoid using magic numbers in your code. The value '10' used for rounding should be defined as a constant with a clear name indicating its purpose.